app/src/main/java/com/dsu/poc/repository/login/ValidationRepository.kt

```diff
@@ -0,0 +1,54 @@
+package com.dsu.poc.repository.login
+
+import android.app.Application
+import android.text.TextUtils
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+
+class ValidationRepository(application: Application) {
+
+    var application: Application
+
+    init {
+        this.application = application
+    }
+
+    fun validateCredentials(userName:String,password:String): LiveData<String> {
+        val loginErrorMessage = MutableLiveData<String>()
+        if(isUserNameValid(userName)){
+            if(!isPasswordValid(password)){
+                loginErrorMessage.value = "Invalid Password"
```

Comment:
no strings should be hardcoded. values should be extracted from strings.xml file